### ISSUE TITLE
Tag new release

### ISSUE THREAD

**Issue by ruudk** [getsentry/sentry-symfony#726]
Would it be possible to tag a new release that includes #723? It would be great to finally not see this deprecation anymore in our CI 🤩 

https://github.com/getsentry/sentry-symfony/compare/4.8.0...master

Thansk 🙏

**Comment by cleptric** [comment]
Yes, we'll tag a new version soon. Please understand that we also have other things to consider after merging a PR, so it might take some time before a release goes out.

Thanks for your understanding.
